packages/app-store/utils.ts

```diff
@@ -60,14 +60,14 @@ function getApps(userCredentials: CredentialData[]) {
     let locationOption: OptionTypeBase | null = null;
 
     /** If the app is a globally installed one, let's inject it's key */
-    if (appMeta.isGlobal) {
-      credentials.push({
-        id: +new Date().getTime(),
-        type: appMeta.type,
-        key: appMeta.key!,
-        userId: +new Date().getTime(),
-      });
-    }
+    // if (appMeta.isGlobal) {
+    //   credentials.push({
+    //     id: +new Date().getTime(),
+    //     type: appMeta.type,
+    //     key: appMeta.key!,
+    //     userId: +new Date().getTime(),
+    //   });
+    // }
```

Comment:
No longer needed if global apps are not defaulted installed

Reply:
Same as my comment above. Also let's just remove instead of commenting.